DESCRIPTION

```diff
@@ -22,10 +23,12 @@ Imports:
   utils,
   withr
 Suggests:
-  arrow,
-  data.table,
-  dplyr,
-  readr,
-  vroom
+    arrow,
+    data.table,
+    dplyr,
+    fst,
+    readr,
+    vroom,
+    covr
```

Comment:
Maybe don't add covr and only install it in the coverage job (or delete the coverage job)?